### PR TITLE
Feature/bump open telemetry

### PR DESCRIPTION
**Pull request by stagnation**

### CODE REVIEW

**Comment by stagnation** [comment]
Need to synchronize all of the deps

**Review by EdSchouten — COMMENTED** [review]
Hey! This got closed?

**Comment by stagnation** [comment]
Yeah, I closed it as it is now just a run-of-the-mill upgrade, where I initially wanted a small targeted incision for OpenTelemetry. And the branch name does not reflect that, so the merge commit would just look out-of-place.

**Comment by stagnation** [comment]
Reformulated in https://github.com/buildbarn/bb-remote-execution/pull/122

File: internal/mock/BUILD.bazel [hunk 1]
```diff
@@ -329,12 +329,18 @@ gomock(
...
+        "Tracer": "BareMockTracer",
+        "TracerProvider": "BareMockTracerProvider",
+    },
     package = "mock",
 )
 
 go_library(
     name = "mock",
     srcs = [
+        "trace_wrap.go",
```
[the remark this answers is not in the document]
Comment (stagnation) [reply internal/mock/BUILD.bazel:343]: Yepp

**Pull request closed without merging**